### PR TITLE
Iht 4332 Removal of createvalidationfunction

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsAnnuityController.scala [hunk 1]
```diff
@@ -33,7 +33,7 @@ trait InsurancePolicyDetailsAnnuityController extends EstateController {
   override val applicationSection = Some(ApplicationKickOutHelper.ApplicationSectionAssetsInsurancePoliciesAnnuities)
   def onPageLoad = authorisedForIht {
     implicit user => implicit request => {
-      estateElementOnPageLoad[InsurancePolicy](insurancePolicyForm, insurance_policy_details_annuity.apply,
+      estateElementOnPageLoad[InsurancePolicy](insurancePolicyAnnuity, insurance_policy_details_annuity.apply,
```
Comment (tyagikumarvineet) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsAnnuityController.scala]: insurancePolicyAnnuity should be renamed to insurancePolicyAnnuityForm

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsInTrustController.scala [hunk 2]
```diff
@@ -32,7 +32,7 @@ trait InsurancePolicyDetailsInTrustController extends EstateController {
 
   def onPageLoad = authorisedForIht {
     implicit user => implicit request => {
-      estateElementOnPageLoad[InsurancePolicy](insurancePolicyForm, insurance_policy_details_in_trust.apply,
+      estateElementOnPageLoad[InsurancePolicy](insurancePolicyInTrust, insurance_policy_details_in_trust.apply,
```
Comment (tyagikumarvineet) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsInTrustController.scala]: insurancePolicyInTrust should be renamed to insurancePolicyInTrustForm

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsMoreThanMaxValueController.scala [hunk 3]
```diff
@@ -34,7 +34,7 @@ trait InsurancePolicyDetailsMoreThanMaxValueController extends EstateController
 
   def onPageLoad = authorisedForIht {
     implicit user => implicit request => {
-      estateElementOnPageLoad[InsurancePolicy](insurancePolicyForm, insurance_policy_details_more_than_max_value.apply,
+      estateElementOnPageLoad[InsurancePolicy](insurancePolicyMoreThanMax, insurance_policy_details_more_than_max_value.apply,
```
Comment (tyagikumarvineet) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsMoreThanMaxValueController.scala]: insurancePolicyMoreThanMax should be renamed to insurancePolicyMoreThanMaxForm

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsPayingOtherController.scala [hunk 4]
```diff
@@ -31,7 +31,7 @@ object InsurancePolicyDetailsPayingOtherController extends InsurancePolicyDetail
 trait InsurancePolicyDetailsPayingOtherController extends EstateController {
   def onPageLoad = authorisedForIht {
     implicit user => implicit request =>
-      estateElementOnPageLoad[InsurancePolicy](insurancePolicyForm, insurance_policy_details_paying_other.apply,
+      estateElementOnPageLoad[InsurancePolicy](insurancePolicyPayingOther, insurance_policy_details_paying_other.apply,
```
Comment (tyagikumarvineet) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsPayingOtherController.scala]: Same as above

File: app/iht/forms/ApplicationForms.scala [hunk 5]
```diff
@@ -197,7 +198,71 @@ object ApplicationForms {
     "moreThanMaxValue" -> optional(boolean)
   )(InsurancePolicy.apply)(InsurancePolicy.unapply)
 
-  val insurancePolicyForm = Form (insurancePolicyMapping)
+  val insurancePolicyDeceasedOwnQuestionForm = Form(
```
Comment (tyagikumarvineet) [inline app/iht/forms/ApplicationForms.scala]: Would be good to have the form validation tests for these new forms

Comment (grantrobson) [inline app/iht/forms/ApplicationForms.scala]: Done

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsPayingOtherController.scala (hunk 4 above)
Comment (grantrobson) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsPayingOtherController.scala]: Done

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsMoreThanMaxValueController.scala (hunk 3 above)
Comment (grantrobson) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsMoreThanMaxValueController.scala]: Done

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsInTrustController.scala (hunk 2 above)
Comment (grantrobson) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsInTrustController.scala]: Done

File: app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsAnnuityController.scala (hunk 1 above)
Comment (grantrobson) [inline app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyDetailsAnnuityController.scala]: Done

**Pull request merged**